### PR TITLE
Styling Leaderboard

### PR DESCRIPTION
**Pull request by carreraprogrammer**
In this branch, I improve the look and feel of the application

### CODE REVIEW

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌
# Hello @carreraprogrammer 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have not attached the linters required for this project. Please attach the required linters and submit it for review again. 👍🏻 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/modules/game.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
+/* eslint-disable */
+// Game
```
Comment (ichala) [inline src/modules/game.js]: - [ ]  linter is used to flag programming errors, bugs, stylistic errors, and suspicious constructs, so to keep your code readable and understandable by everyone, kindly remove the eslint disabling command and please fix all the linters error 🙏

File: src/modules/display-scores.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
+/* eslint-disable linebreak-style */
```
Comment (ichala) [inline src/modules/display-scores.js:1]: - [ ]  linter is used to flag programming errors, bugs, stylistic errors, and suspicious constructs, so to keep your code readable and understandable by everyone, kindly remove the linters disabling command and please fix all the linters error 🙏

File: src/modules/add-score.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```
Comment (ichala) [inline src/modules/add-score.js:1]: - [ ]  linter is used to flag programming errors, bugs, stylistic errors, and suspicious constructs, so to keep your code readable and understandable by everyone, kindly remove the linters disabling command and please fix all the linters error 🙏

File: src/index.js [hunk 4]
```diff
@@ -1,6 +1,8 @@
+/* eslint-disable linebreak-style */
```
Comment (ichala) [inline src/index.js:1]: - [ ]  linter is used to flag programming errors, bugs, stylistic errors, and suspicious constructs, so to keep your code readable and understandable by everyone, kindly remove the linters disabling command and please fix all the linters error 🙏

File: src/modules/leaderboard-API.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```
Comment (ichala) [inline src/modules/leaderboard-API.js:1]: - [ ]  linter is used to flag programming errors, bugs, stylistic errors, and suspicious constructs, so to keep your code readable and understandable by everyone, kindly remove the linters disabling command and please fix all the linters error 🙏

File: src/style.css [hunk 6]
```diff
@@ -1,4 +1,4 @@
-/******** Reset *********/
+/* stylelint-disable length-zero-no-unit */
```
Comment (ichala) [inline src/style.css]: - [ ]  linter is used to flag programming errors, bugs, stylistic errors, and suspicious constructs, so to keep your code readable and understandable by everyone, kindly remove the linters disabling command and please fix all the linters error 🙏

File: src/style.css [hunk 7]
```diff
@@ -53,12 +72,14 @@ body {
   border: 2px solid black;
   padding: 5px;
   cursor: pointer;
-  transition: all 1s ease-in-out;
+  /* stylelint-disable-next-line csstree/validator */
```
Comment (ichala) [inline src/style.css:73]: - [ ]  linter is used to flag programming errors, bugs, stylistic errors, and suspicious constructs, so to keep your code readable and understandable by everyone, kindly remove the linters disabling command and please fix all the linters error 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi  @carreraprogrammer 
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great Design✔️ 
- No linters Errors ✔️

 ## Required Changes ♻️

- [ ] 🛑  Kindly Update the readme file to make it more professional by following these steps :

1. kindly update the setup, install & Getting Started sections where you guide the user on how to clone your application, how to install it, and how to run it locally 
2. kindly update your live demo link if you have one or remove the section
3. kindly remove any unused sections in your readme file
4. your  MIT Licence link is broken , kindly try to fix it and update it

- [ ] 🛑  You have some errors in your application kindly try to fix them :

image


- [ ] 🛑  **_Check The comments below_**



## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @carreraprogrammer 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Well done making the required changes the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**